src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.html

```diff
@@ -18,6 +18,7 @@
              aria-labelledby="username"
              class="form-control pl-4"
              required
+             autocomplete="off"
```

Comment:
I don't think we should disable autocomplete on the login page, since login is the usual purpose of autocomplete.

Reply:
@votdev Is there a document, guideline or anything saying that auto-completion should not be allowed on login forms? - If yes we may consider it.

Reply:
I removed it because [browsers will ignore it on login pages](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#The_autocomplete_attribute_and_login_fields).